### PR TITLE
Utilities price list + парочка тестов для банка

### PR DESCRIPTION
**Pull request by Spec163**
UtilitiesPriceList components (int client-ui module) is already reviewed.

### CODE REVIEW

**Comment by kostua16** [comment]
Depends on #266 due to build failure for commit fcc5239c6385807dd8d46355c1b5cd8c13ad10cb

**Comment by kostua16** [comment]
Build successfully completed for commit 6a990be6ff346cd71cecf3c64e3696fd399201e2

**Comment by kostua16** [comment]
Build successfully completed for commit 8c82288a5b7e2557b14e92c6cec989986f08b6e5

**Comment by kostua16** [comment]
Build successfully completed for commit 8c82288a5b7e2557b14e92c6cec989986f08b6e5

**Comment by kostua16** [comment]
Build successfully completed for commit 8c82288a5b7e2557b14e92c6cec989986f08b6e5

**Comment by kostua16** [comment]
Build successfully completed for commit 85e99d0a553bb9662d60bb713674dec7ed338952

**Comment by kostua16** [comment]
Build successfully completed for commit 9f836ca99421db9018e20aa84902d80bdf53425c

**Comment by kostua16** [comment]
Build successfully completed for commit 51d0feef8107f6c1b1b0b441be8a8189f01b363c

**Comment by kostua16** [comment]
Build successfully completed for commit 51d0feef8107f6c1b1b0b441be8a8189f01b363c

**Comment by kostua16** [comment]
Build successfully completed for commit 51d0feef8107f6c1b1b0b441be8a8189f01b363c

**Comment by kostua16** [comment]
Build successfully completed for commit 9d5c3f183f367b4e3bbb9e1b8d0e92a11533236f

**Comment by kostua16** [comment]
Build successfully completed for commit 729023380634bd5085dcfbe825e2863d5e4e9e6c

File: services/bank/src/docs/asciidoc/bank.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
+= Bank API Guide
+Konstantin Leontiev;
```
Comment (kostua16) [inline services/bank/src/docs/asciidoc/bank.adoc:2]: Correct author

**Comment by kostua16** [comment]
Build successfully completed for commit f64cdb7b16ad14b8699a7d837e15a5a8ad3fe99f

File: services/gibdd/src/docs/asciidoc/gibdd.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-= Communal API Guide
+= Gibdd API Guide
 Konstantin Leontiev;
```
Comment (kostua16) [inline services/gibdd/src/docs/asciidoc/gibdd.adoc:2]: Correct Author

**Review by kostua16 — CHANGES_REQUESTED** [review]

**Comment by kostua16** [comment]
Build successfully completed for commit 74530d5b290180a5a911712188f8d5de77babfcb

File: services/gibdd/src/docs/asciidoc/gibdd.adoc (hunk 2 above)
Comment (Spec163) [reply services/gibdd/src/docs/asciidoc/gibdd.adoc:2]: В этом файле не стал менять, тк ГИБДД это сервис кураторов

**Comment by kostua16** [comment]
Build successfully completed for commit 66508d8ecef1d96862ce1b293fdeda586e6a3fe6

**Review by kostua16 — APPROVED** [review]

**Pull request merged**